README.md

```diff
@@ -24,7 +24,8 @@ const combinedComposables = composable0.compose(composable1, composable2, compos
 
 ### Descriptor
 
-**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain the information necessary to create an object instance by a *composable*.
+**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain
+the information necessary to create an object instance by a *composable*.
```

Comment:
I realize this is frustrating to you, but hard line breaks in markdown files literally break every display except those that are *at least as wide* as whatever arbitrary number you decide to break the lines at, and for windows that are wider than the arbitrary number, hard line breaks fail to take advantage of the available space.

In other words, it breaks basically everybody's experience except the author's, as demonstrated above.

> Only 1 of 9 people is using soft wraps.

1. That's anecdotal. Definitely not a good enough reason to break the very commonly used diff experience.
2. If you like hard wraps, use hard wraps in your own personal projects.